Tensile/Source/lib/source/ContractionSolution.cpp

```diff
@@ -1604,6 +1604,8 @@ namespace Tensile
     size_t ContractionSolution::getSKGrid(Hardware const& hardware, size_t tiles) const
     {
         AMDGPU const* pAMDGPU = dynamic_cast<AMDGPU const*>(&hardware);
+        if(pAMDGPU->skFixedGrid > 0)
```

Comment:
Don't we need max value check below (skMaxCUs or tiles)?
In other words, can we use any skFixedGrid value without limitation?

Reply:
Yea this overrides all other options so we can test grids both larger and smaller than CU count. This option makes it easier to script benchmarking sweeps to analyze performance at different grid sizes.

Reply:
OK. Thanks for confirmation.